R/license.R

```diff
@@ -87,6 +87,18 @@ use_cc0_license <- function(name = find_name()) {
   use_license_template("cc0", name)
 }
 
+#' @rdname licenses
+#' @export
+use_ccby_license <- function(name = find_name()) {
+  force(name)
+  check_is_package("use_ccby_license()")
+
+  use_description_field("License",
+                        "CCBY-4.0",
+                        overwrite = TRUE)
```

Comment:
Can you use styler on the bits you're changing? Re: indentation and line breaks, specifically.

Reply:
I did this with the styler adding, but it looks like the style is inconsistent with other functions in license.R - did you want me to run it on the whole license.R file?

Reply:
Sure it's fine to style this whole file.